curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md

```diff
@@ -548,3 +548,444 @@ Which of the following is the correct way to use the `minmax()` function?
+
+#### --text--
+
+What is the effect of `grid-template-columns: 1fr 2fr 1fr`?
+
+#### --distractors--
+
+Creates three equal-width columns.
+
+---
+
+Makes the middle column twice as wide as the others.
+
+---
+
+Forces all columns to be exactly 1fr wide.
+
+#### --answer--
+
+Creates three columns where the middle is twice as wide as the sides.
+
+### --question--
+
+#### --text--
+
+Which property would you use to override alignment for a specific grid item?
+
+#### --distractors--
+
+`grid-item-align`
+
+---
+
+`align-self`
+
+---
+
+`item-placement`
+
+#### --answer--
+
+`justify-self`
+
+### --question--
+
+#### --text--
+
+What does `repeat(3, minmax(100px, 1fr))` create?
+
+#### --distractors--
+
+Three columns that can't shrink below 100px.
+
+---
+
+Three fixed 100px columns.
+
+---
+
+Three rows with maximum height of 1fr.
+
+#### --answer--
+
+Three columns that grow proportionally but won't shrink below 100px.
+
+### --question--
+
+#### --text--
+
+How does `auto-fill` differ from `auto-fit` in grid layouts?
+
+#### --distractors--
+
+`auto-fill` creates more tracks than needed.
+
+---
+
+`auto-fit` only works with fixed sizes.
+
+---
+
+`auto-fill` ignores the minmax function.
+
+#### --answer--
+
+`auto-fill` creates empty tracks while `auto-fit` collapses them.
+
+### --question--
+
+#### --text--
+
+Which property controls the order of grid items visually?
+
+#### --distractors--
+
+`grid-order`
+
+---
+
+`item-position`
+
+---
+
+`flow-order`
+
+#### --answer--
+
+`order`
+
+### --question--
+
+#### --text--
+
+What is the purpose of the `subgrid` value?
+
+#### --distractors--
+
+To create nested grids with independent tracks.
+
+---
+
+To make a grid item inherit its parent's track sizing.
+
+---
+
+To automatically fill empty grid areas.
+
+#### --answer--
+
+To align a nested grid with its parent's tracks.
+
+### --question--
+
+#### --text--
+
+How can you create asymmetric grid layouts?
+
+#### --distractors--
+
+By using only `fr` units.
+
+---
+
+By mixing different length units in `grid-template-columns`.
+
+---
+
+By setting `grid-asymmetric: true`.
+
+#### --answer--
+
+By defining different sizes for each track.
+
+### --question--
+
+#### --text--
+
+What does `grid-column-start: 2` do to a grid item?
+
+#### --distractors--
+
+Makes it span 2 columns.
+
+---
+
+Offsets it by 2 pixels.
+
+---
+
+Positions it starting at the second vertical grid line.
+
+#### --answer--
+
+Makes it start at the second column line.
+
+### --question--
+
+#### --text--
+
+Which property would you use to control overflow behavior in grid tracks?
+
+#### --distractors--
+
+`grid-overflow`
+
+---
+
+`track-sizing`
+
+---
+
+`fit-content`
+
+#### --answer--
+
+`minmax()`
+
+### --question--
+
+#### --text--
+
+How do you make grid items align to the end of their grid area vertically?
+
+#### --distractors--
+
+`align-grid: end;`
+
+---
+
+`justify-items: flex-end;`
+
+---
+
+`grid-align: bottom;`
+
+#### --answer--
+
+`align-items: end;`
```

Comment:
same issue here

was this covered in the lectures or workshop? I know that `align-items` was covered but I don't know if the `end` value was taught. I haven't gone through the lectures, one by one. But it is not on the review page. So either the review page needs to be updated, which is a separate issue. Or we need to update this question to something else.